src/genome.py

```diff
@@ -46,7 +48,9 @@ def copy_te(self, te: int, offset: int) -> int | None:
 
         If te is not active, return None (and do not copy it).
         """
-        ...  # not implemented yet
+        
+        self.copy_te(te, offset)
```

Comment:
You _really_ shouldn't do this! It is an infinite loop, if the other classes didn't override the methods